### PR TITLE
Primality test implemented in Dart - #2111 resolved

### LINKED ISSUE #2111: AKS primality test
- [x] C
- [x] C++
- [x] C# - @Yashank18
- [ ] CoffeeScript
- [ ] Dart - @Rukmini-Meda
- [ ] Go
- [x] Java
- [x] JS
- [ ] Kotlin
- [ ] Php
- [x] Python
- [x] Readme - @goyalmuskan
- [ ] Ruby
- [ ] Rust
- [ ] TypeScript - @Kuljeet-123

### PR DESCRIPTION
**Pull request by Rukmini-Meda**
Fixes #2111 

#### Checklist:
- [x] 4 space indentation.
- [x] Coding conventions are followed.
- [x] Input is taken dynamically.
- [x] Sample Input / Output is added at the end of file.
- [x] Logic Documentation (Comments).
- [x] File names are correct.

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

#### Changes proposed in this pull request:
- AKS primality test is implemented in Dart

#### Languages Used:
- Dart

#### Files Added:
- AKS.dart

> We're happy to help you get this ready -- don't be afraid to ask for help.

Thanks!

### CODE REVIEW

**Comment by Rukmini-Meda** [comment]
@asha15 @MastersAbh @SakshiShreya @singh-shreya6 @raghav-dalmia Please review and merge this PR.

File: AKS_Primarility_Test/AKS.dart [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+// Function to find the primality of a given number using AKS primality test
+bool aks_primality(int number){
+    List c = List(100);
+    coef(number, c);
+    c[0] ++;
+    c[number] --;
+    int i = number;
+    while ((i --) > 0 && c[i] % number == 0);
+    return i < 0;
+}
```
Comment (asha15) [inline AKS_Primarility_Test/AKS.dart:24]: keep a line space after this

File: AKS_Primarility_Test/AKS.dart [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+// Driver method of the program
+void main(){
+    // Input of the number
+    print("Enter a number for primality test:");
+    var input = stdin.readLineSync();
+    int number = int.parse(input);
```
Comment (asha15) [inline AKS_Primarility_Test/AKS.dart:30]: keep a line space after this

**Comment by Rukmini-Meda** [comment]
@asha15 I have made the changes. Please merge further.

**Pull request merged**